index.bs

```diff
@@ -198,7 +215,13 @@ The <dfn method for="StorageBucketManager">keys()</dfn> method steps are:
 
     1. Let |keys| be a new [=/list=].
 
-    1. For each |key| in |shelf|'s [=bucket map=], [=list/append=] |key| to |keys|.
+    1. For each |key| in |shelf|'s [=bucket map=], run the following steps:
+
+        1. Let |bucket| be |shelf|'s [=/bottle map=][|key|].
```

Comment:
Should this be `[=/bucket map=][|key|]`, because I think we want the bucket that holds the bottle map?

Reply:
yes, done.